modules/gcal/widget.go

```diff
@@ -61,27 +53,5 @@ func (widget *Widget) fetchAndDisplayEvents() {
 		widget.calEvents = calEvents
 	}
 
-	widget.app.QueueUpdateDraw(func() {
-		widget.display()
-	})
-}
-
-func updateLoop(widget *Widget) {
```

Comment:
This seemed unnecessary, as the refresh of the widget itself should handle this, no?

Reply:
Yeah, I'm not sure why that's there. It should be removed.